content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -2,127 +2,143 @@
-  message will appear if creation of a vendor bill is attempted without receiving anything.
+  message appears if creation of a vendor bill is attempted without receiving anything.
 
-.. image:: control_bills/bill-control-policy-error-message.png
-   :align: center
-   :alt: Bill control policy draft bill error message.
+  .. image:: control_bills/control-bills-error-message-popup.png
+     :align: center
+     :alt: Bill control policy draft bill error message.
 
 .. note::
-   If one or two products need a different control policy, the default bill control setting can be
-   overridden by going to the :guilabel:`Purchase` tab in a product's template and modifying its
-   :guilabel:`Control Policy` field.
-
-Example flow: Ordered quantities
---------------------------------
+   If a specific product should use a different control policy than selected in the *Purchase* app
+   settings, the *bill control* policy for that product can be changed from its product form.
 
-To complete an example workflow using the *ordered quantities* bill control policy, first go to
-:menuselection:`Purchase --> Configuration --> Settings`, scroll down to the :guilabel:`Invoicing`
-section, and select :guilabel:`Ordered quantities`. Then, :guilabel:`Save` changes.
-
-In the :guilabel:`Purchase` app, create a new :abbr:`RFQ (Request for Quotation)`. Fill out the
-information on the quotation form, add products to the invoice lines, and click :guilabel:`Confirm
-Order`. Then, click :guilabel:`Create Bill`. Since the policy is set to *ordered quantities*, the
-draft bill can be confirmed as soon as it is created, without any products actually being received.
-
-Example flow: Received quantities
----------------------------------
-
-To complete an example workflow using the *received quantities* bill control policy, first go to
-:menuselection:`Purchase --> Configuration --> Settings`, scroll down to the :guilabel:`Invoicing`
-section, and select :guilabel:`Received quantities`. Then, :guilabel:`Save` changes.
-
-In the :guilabel:`Purchase` app, create a new :abbr:`RFQ (Request for Quotation)`. Fill out the
-information on the quotation form, add products to the invoice lines, and click :guilabel:`Confirm
-Order`. Then, click on the :guilabel:`Receipt smart button`. Set the quantities in the
-:guilabel:`Done` column to match the quantities in the :guilabel:`Demand` column, and
-:guilabel:`Validate` the changes. Then, in the purchase order, click :guilabel:`Create Bill` and
-:guilabel:`Confirm`. Since the policy is set to *received quantities*, the draft bill can be
-confirmed *only* when at least some of the quantities are received.
+   Navigate to :menuselection:`Purchase app --> Products --> Products`, and select a product. From
+   the product form, click the :guilabel:`Purchase` tab. Under the :guilabel:`Vendor Bills` section,
+   modify the selection in the :guilabel:`Control Policy` field, and :guilabel:`Save` changes.
```

Comment:
```suggestion
   modify the selection in the :guilabel:`Control Policy` field.
```
With 17, this would auto-save, correct? Since it's being done on a Product Form, and not on the Settings page...?